deepdiff/deepdiff.py

```diff
@@ -343,11 +343,17 @@ class DeepDiff(RemapDict):
 
     show_warning = True
 
-    def __init__(self, t1, t2, ignore_order=False, report_repetition=False, significant_digits=None, **kwargs):
+    def __init__(self, t1, t2,
+                 ignore_order=False, report_repetition=False, significant_digits=None,
+                 exclude_paths=set(), exclude_types=set(), **kwargs):
         if kwargs:
-            raise ValueError("The following parameter(s) are not valid: %s\nThe valid parameters are ignore_order, report_repetition and significant_digits" % ', '.join(kwargs.keys()))
+            raise ValueError(("The following parameter(s) are not valid: %s\n" +
+                             "The valid parameters are ignore_order, report_repetition, significant_digits," +
+                             "exclude_paths and exclude_types.") % ', '.join(kwargs.keys()))
         self.ignore_order = ignore_order
         self.report_repetition = report_repetition
+        self.exclude_paths = exclude_paths
```

Comment:
I think we should force convert the `exclude_paths` to set here. Because most people will pass it as list.